noxfile.py

```diff
@@ -100,7 +100,11 @@ def lint(session):
 def test(session):
     session.install("-r", "requirements-dev.txt")
     session.run("python", "-m", "eland.tests.setup_tests")
+    session.run("python", "setup.py", "install")
     session.run("pytest", "--doctest-modules", *(session.posargs or ("eland/",)))
+    session.run(
```

Comment:
Let's add a check that sees if `session.posargs` aren't empty and then only if they are empty do we run the `--nbval` or `eland/tests/ml` below, that way if you're targeting a single test with `nox -rs test-3.8 -- eland/tests/test_...` then ONLY those tests are executed.

Does this make sense?

Reply:
I only wanted to execute the notebook tests. So, Ill remove the `session.posargs`
Sorry, I really don't know what `session.posargs` does.

Reply:
Oh no problem, `session.posargs` is essentially a tuple of string arguments that can be passed in via `nox -s <session> -- <posargs>`

for example if I were to run `nox -s test-3.8 -- tests/a tests/b` then `session.posargs` would be `("tests/a", "tests/b")`

The feature isn't super-well documented on the Nox side unfortunately but very useful for test suites especially.

Reply:
Yeah  thanks